extensions/catalog/federated-catalog-cache/src/main/java/org/eclipse/dataspaceconnector/catalog/cache/controller/FederatedCatalogApiController.java

```diff
@@ -29,8 +29,8 @@
 
 import java.util.Collection;
 
-@Consumes({MediaType.APPLICATION_JSON})
-@Produces({MediaType.APPLICATION_JSON})
+@Consumes({ MediaType.APPLICATION_JSON })
```

Comment:
I noticed that everyone has their taste in fact of whitespaces on arrays, why don't avoid curly braces since we only produce/consume only json?
Like
```
@Consumes(MediaType.APPLICATION_JSON)
@Produces(MediaType.APPLICATION_JSON)
```

Reply:
Those sorts of things should be handled by the IntelliJ code formatter. The one i'm using is located at `resources/edc-codestyle.xml`.

Reply:
yeah, I don't care that much too, but looking at the history seems like it's not the first time that this changes, just to avoid ping-pong changelogs :)